### PR TITLE
Handle GET/POST experience requests

### LINKED ISSUE #20: Implement API route to return all "Experiences"
On the `/resume/experience` route, you should:

- [ ] Return the Experience list when a GET request is made. The return should be in a list format.

### LINKED ISSUE #21: Implement API route to add new "Experience"
On the `/resume/experience` route, you should:

- [ ] Return the index position of the new Experience in the Experience list when a POST request is made

### PR DESCRIPTION
**Pull request by jjoeldaniel**
## Description

- [x] Return the Experience list when a GET request is made. The return should be in a list format.
- [x] Return the index position of the new Experience in the Experience list when a POST request is made
- [x] Pass existing experience unit tests

Resolves #20 
Resolves #21

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -44,11 +44,32 @@ def experience():
...
     Handle experience requests
     '''
-    if request.method == 'GET':
-        return jsonify()
 
-    if request.method == 'POST':
-        return jsonify({})
+    if request.method == "GET":
+        experiences: list[Experience] = data.get("experience", [])
+        if index := request.args.get("index") is not None:
```
Comment (Pradyuman7) [inline app.py:50]: Do the listed issues talk about this update? I think this might be relevant for some other issue.

**Review by Pradyuman7 — COMMENTED** [review]
Can we also have some tests for this change (unless there are separate issues for that)

**Comment by jjoeldaniel** [comment]
> Can we also have some tests for this change (unless there are separate issues for that)

I've deliberately not included any handling of edge cases and only intended to pass the current experience unit tests, as some issues such as #12 seem to address similar issues. I am planning to make a new issue to add/pass new unit tests.

**Review by Pradyuman7 — APPROVED** [review]
LGTM 🚀
